src/Web/Rsp.IrasPortal.Web/Controllers/QuestionSetController.cs

```diff
@@ -1,287 +1,215 @@
 ﻿using System.Data;
 using System.Diagnostics.CodeAnalysis;
-using System.Text;
-using ExcelDataReader;
+using System.Text.Json;
 using FluentValidation;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Rsp.IrasPortal.Application.Constants;
 using Rsp.IrasPortal.Application.DTOs;
 using Rsp.IrasPortal.Application.Services;
+using Rsp.IrasPortal.Web.Extensions;
 using Rsp.IrasPortal.Web.Models;
+using static Rsp.IrasPortal.Application.Constants.QuestionCategories;
 
 namespace Rsp.IrasPortal.Web.Controllers;
 
 [ExcludeFromCodeCoverage]
 [Route("[controller]/[action]", Name = "questionset:[action]")]
```

Comment:
Not a big deal, but I have kept it to 3 letter prefix when defining RouteName
```suggestion
[Route("[controller]/[action]", Name = "qsc:[action]")] //qsc = QuestionSetController
```